Framework/API/inc/MantidAPI/AlgoTimeRegister.h

```diff
@@ -27,31 +28,36 @@ class AlgoTimeRegister {
   struct Info {
     std::string m_name;
     std::thread::id m_threadId;
-    timespec m_begin;
-    timespec m_end;
+    Kernel::time_point_ns m_begin;
+    Kernel::time_point_ns m_end;
 
-    Info(const std::string &nm, const std::thread::id &id, const timespec &be, const timespec &en)
+    Info(const std::string &nm, const std::thread::id &id, const Kernel::time_point_ns &be,
+         const Kernel::time_point_ns &en)
         : m_name(nm), m_threadId(id), m_begin(be), m_end(en) {}
   };
 
   class Dump {
     AlgoTimeRegister &m_algoTimeRegister;
     timespec m_regStart;
+    Kernel::time_point_ns m_regStart_chrono;
```

Comment:
I like the switch to `std::chrono` - Could you help me undertand the difference between `m_regStart` and `m_regStart_chrono` and whether we need the former?

Reply:
Good catch! All of the `timespec` stuff should have been removed.